src/components/pill-selector-dropdown/PillSelector.js

```diff
@@ -54,6 +55,7 @@ class PillSelector extends React.Component<Props, State> {
         error: '',
         inputProps: {},
         placeholder: '',
+        // $FlowFixMe defaultProps does not propagate to forwardRef components
```

Comment:
Curious about this...according to this Flow article:

https://medium.com/flow-type/supporting-react-forwardref-and-beyond-f8dd88f35544

It seems `forwardRef` support was added in version `0.89` and looking at `flow-bin` we're running at least `0.95`?

Anyway we can get Flow to work with the `forwardRef`?

Reply:
That article is helpful. Thanks. I managed to get it to work!